v2.0 compile error in test

On v2.0-branch, with `--enable-tests `, the tests fail to compile with GCC >= 6.  This is due to libva deprecated warnings and the tests are compiled with `-Wall -Werror`.  Older GCC versions do not appear to treat deprecated-declarations as errors even with `-Wall -Werror`.

```
g++ -DHAVE_CONFIG_H -I. -I../src  -I/usr/include/libdrm -I/home/uaeoff/Work/workspace/media/install/include -I/home/uaeoff/Work/workspace/media/install/include -DGTEST_HAS_PTHREAD=1 -DGTEST_USE_OWN_TR1_TUPLE=0 -DGTEST_LANG_CXX11=1 -DGTEST_HAS_TR1_TUPLE=1 -DPTHREADS -DVA_DRIVERS_PATH="\"/home/uaeoff/Work/workspace/media/install/lib/dri\"" -DTEST_VA_DRIVERS_PATH="\"/home/uaeoff/Work/workspace/media/build/intel-vaapi-driver/src/.libs\"" -std=c++11   -I/home/uaeoff/Work/workspace/media/install/include -Wall -Werror -I../test/gtest/include   -g3 -MT test_i965_drv_video-i965_avcd_config_test.o -MD -MP -MF .deps/test_i965_drv_video-i965_avcd_config_test.Tpo -c -o test_i965_drv_video-i965_avcd_config_test.o `test -f 'i965_avcd_config_test.cpp' || echo './'`i965_avcd_config_test.cpp
i965_avcd_config_test.cpp:99:7: error: ‘VAProfileH264Baseline’ is deprecated [-Werror=deprecated-declarations]
{ VAProfileH264Baseline, VAEntrypointVLD, &ProfileNotSupported },
^~~~~~~~~~~~~~~~~~~~~
In file included from test.h:32:0,
from i965_test_environment.h:28,
from i965_test_fixture.h:28,
from i965_config_test.h:28,
from i965_avcd_config_test.cpp:25:
/home/uaeoff/Work/workspace/media/install/include/va/va.h:327:5: note: declared here
VAProfileH264Baseline va_deprecated_enum = 5,
^~~~~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
Makefile:607: recipe for target 'test_i965_drv_video-i965_avcd_config_test.o' failed
make[1]: *** [test_i965_drv_video-i965_avcd_config_test.o] Error 1
make[1]: Leaving directory '/home/uaeoff/Work/workspace/media/build/intel-vaapi-driver/test'
Makefile:420: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -36,7 +36,7 @@ before_install:
   - sudo apt-get install -y intel-gpu-tools
  
 install:
-  - git clone https://github.com/01org/libva.git
+  - git clone -b v2.0-branch https://github.com/01org/libva.git
   - (cd libva && ./autogen.sh && ./configure --prefix=/usr && sudo make install)
 
 addons:

--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -102,6 +102,7 @@ test_i965_drv_video_CPPFLAGS =						\
 
 test_i965_drv_video_CXXFLAGS =						\
 	-Wall -Werror							\
+	-Wno-deprecated-declarations					\
 	$(AM_CXXFLAGS)							\
 	$(NULL)
 

